Makefile

```
@@ -1,11 +1,12 @@
-.PHONY: lint format dev-setup dev-install prod-install
+.PHONY: lint format dev-setup dev-install prod-install test install-test-requirements
 
 # Commands
 ## Python
 flake8	= flake8 python/fastexcel *.py
 isort	= isort python/fastexcel *.py
 black	= black python/fastexcel *.py
 mypy	= mypy python/fastexcel *.py
+pytest	= python -m pytest -vvv -s python/tests
```

```suggestion
pytest	=  pytest -vvv -s python/tests
```
doesn't do the job ? 🤔

---

Depending on the dev's shell and how its path hashing/compression works,  it can lead to hard-to-spot behaviours. With `python -m` we're sure the right one is called